test/puppetlabs/pcp/messaging_test.clj

```diff
@@ -170,26 +165,3 @@
     (with-app-with-config app broker-services broker-config
       (client/wait-for-connection client (* 40 1000))
       (is (client/connected? client) "Should be reconnected"))))
-
-(deftest association-checkers-test
-  (with-app-with-config app broker-services broker-config
-    (with-open [client (connect-client "client01" (constantly true))]
-      (is (= client (client/wait-for-association client (* 40 1000))))
-      (is (= false (client/associating? client)))
-      (is (= true (client/associated? client))))))
-
-(deftest connect-with-too-small-message-size
```

Comment:
This test should be kept, and updated to explicitly send a message to itself (via the broker) to test, rather than relying on the association response to trigger the error.